xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-test/xwiki-platform-panels-test-docker/src/test/it/org/xwiki/panels/test/ui/docker/PanelIT.java

```diff
@@ -73,71 +86,92 @@ public void verifyApplicationsPanelEntry()
      * @see "XWIKI-8591: Cannot use a panel with a name containing spaces"
      */
     @Test
-    public void addPanelWithSpacesAndSymbolsInName()
+    @Order(2)
+    void addPanelWithSpacesAndSymbolsInName(TestUtils testUtils, TestReference testReference)
     {
+        String testMethodName = testReference.getLastSpaceReference().getName();
+        String testClassName = testReference.getSpaceReferences().get(0).getName();
+
         // Delete panel that will be created by the test
-        String panelName = "Is # & \u0163 triky\"? c:\\windows /root $util";
-        getUtil().deletePage("Panels", panelName);
+        String panelName = SPECIAL_TITLE;
+        testUtils.deletePage("Panels", panelName);
 
         // Create a panel whose name contain spaces.
         PanelEditPage panelEditPage = PanelsHomePage.gotoPage().createPanel(panelName);
 
         // We cannot reuse the panel name since we need special escapes.
         panelEditPage.setContent(
-            String.format(PanelEditPage.DEFAULT_CONTENT_FORMAT, "Is # & \\u0163 triky\\\"? c:\\\\windows /root $util",
-                getTestMethodName()));
+            String.format(PanelEditPage.DEFAULT_CONTENT_FORMAT, SPECIAL_CONTENT,
+                testMethodName));
         panelEditPage.clickSaveAndContinue();
 
+        // Checks that the new panel is listed in the Panels home live data.
+        PanelsHomePage panelsHomePage = PanelsHomePage.gotoPage();
+        LiveDataElement liveData = panelsHomePage.getLiveData();
+        liveData.waitUntilHasContentReady();
+        liveData.filterColumn(1, panelName);
+        liveData.waitUntilRowCountEqualsTo(1);
+        liveData.hasRow("Description", "Panel Description");
+        liveData.hasRow("Panel type", "view");
+        liveData.hasRow("Category", "Information");
```

Comment:
I agree about the layout, While I agree the first option look better, I don't think it is easy to implement, especially if we consider adding new layouts in the future. 
I'll refactor `LiveDataElement` to only provide operations to access the layout POs, and to switch between them, and move the operations existing operations in a new `TableLayoutElement` PO.

As for the wait at the end of the filter, I'm not waiting by default because one can update multiple filters without wanting to wait for the LD to reload before typing the next filter.
If you think it is less error-prone, I can introduce a `wait` parameter, that way, filter on three column would become:
```
liveData.filterColumn(1, "value1", false); // does not wait
liveData.filterColumn(2, "value2", false); // does not wait
liveData.filterColumn(3, "value3"); // waits once all the filters are set
```

Reply:
> I'll refactor LiveDataElement to only provide operations to access the layout POs, and to switch between them, and move the operations existing operations in a new TableLayoutElement PO.

I think the menu and the pagination can stay in ``LiveDataElement``, while the rest needs to be moved to the layout PO yes.

> I can introduce a wait parameter

+1, with wait=true by default.